lib/check-minor-version.js

```diff
@@ -0,0 +1,16 @@
+var semver = require('semver')
+
+module.exports = function () {
+  require('electron-version')(function (err, electronVersion) {
```

Comment:
Not sure how `electron-version` works, but `Application` supports a `path` on creation so I would guess that path would need to be threaded through to `electron-version` so that it finds it in the correct place.